frontend/README.md

```diff
@@ -6,7 +6,7 @@
 
 - [AngularJS](./angular-js/)
 
-- [React](../react/)
+- [React](./react/README.md)
```

Comment:
I think you don't need the /README.md, just keep the:
```
- [React](./react/)
```

like the rest of the techs